scripts/health_check.sh

```
@@ -0,0 +1,35 @@
+            SUCCESS_COUNT=$((SUCCESS_COUNT + 1))
+        else
+            echo "Health check failed, resetting success count"
+            SUCCESS_COUNT=0
+        fi
+        sleep 5
+        COUNTER=$((COUNTER + 1))
+    done
+
+    if [ $SUCCESS_COUNT -lt $REQUIRED_SUCCESSES ]; then
+        echo "Deployment failed: Service did not pass $REQUIRED_SUCCESSES consecutive health checks within 60 seconds"
+        docker stop ukhsc-system-backend-api-$target_env || true
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)